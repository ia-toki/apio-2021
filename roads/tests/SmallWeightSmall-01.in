c10234cabdfd6eae47773fb6f163e0350fc797e1
101
68 52 6
99 49 9
67 12 5
96 89 6
43 55 5
37 7 10
27 64 10
20 57 6
0 40 9
45 57 8
14 91 8
82 47 7
19 35 8
46 15 8
55 96 6
34 52 7
13 1 9
26 61 8
84 11 7
58 51 5
78 32 6
31 62 7
29 78 7
18 44 7
5 63 8
76 9 8
21 100 9
44 43 6
4 42 8
73 62 10
50 13 7
80 72 6
65 97 5
66 44 9
62 13 6
77 73 10
51 81 9
100 83 10
63 78 9
60 66 10
52 96 8
17 32 7
98 24 9
57 14 7
61 21 5
33 72 9
94 38 10
95 38 6
9 39 8
69 63 9
86 97 8
36 92 7
2 20 8
75 53 9
54 95 9
89 78 8
70 61 7
11 4 8
71 0 6
3 98 5
49 33 7
59 66 6
15 31 6
74 94 9
23 63 9
47 79 7
6 12 8
1 100 8
40 80 9
79 56 8
38 99 6
87 21 10
83 54 8
92 89 10
16 21 7
85 43 9
25 0 9
28 4 8
91 48 9
41 81 7
72 57 8
12 43 7
35 6 6
42 75 7
10 7 5
30 63 8
32 82 10
22 64 7
8 22 6
39 18 7
53 82 8
48 90 7
93 97 5
64 100 9
97 60 10
7 40 7
81 91 7
24 44 7
90 19 9
88 11 7
