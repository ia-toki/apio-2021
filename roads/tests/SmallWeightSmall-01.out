064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
764 423 159 43 8 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
